packages/mongo-nav/src/MongoNav.spec.tsx

```diff
@@ -165,13 +170,13 @@ describe('packages/mongo-nav', () => {
 
     test('when the reloadData function is called, MongoNav refetches data', async () => {
       // twice, once for main data and once for alerts polling
-      expect(fetchMock).toHaveBeenCalledTimes(2);
+      expect(fetchMock).toHaveBeenCalledTimes(1);
```

Comment:
comment no longer applies. shouldn't the number of calls be zero since it was reset?